flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "skip-arches": [
+    "arm"
```

Comment:
This isn't available drop this file. x86_64 and aarch64 are the only two available

Reply:
sorry, this is legacy and I will remove